packer/install-dependencies.sh

```diff
@@ -361,19 +361,25 @@ K8S_VERSIONS="
 1.12.7
 1.12.6
 1.11.9
+1.11.9-azs
 1.11.8
+1.11.8-azs
 1.10.13
 1.10.12
 1.9.11
 1.9.10
 "
 for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
-    HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
+    if [[ $KUBERNETES_VERSION == *"azs"* ]]; then
+      HYPERKUBE_URL="msazurestackdocker/hyperkube-amd64:v${KUBERNETES_VERSION}"
+    else
+      HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
+      CONTAINER_IMAGE="k8s.gcr.io/cloud-controller-manager-amd64:v${KUBERNETES_VERSION}"
```

Comment:
I think all of this should be outside of the if block, right? The only conditional change is to `HYPERKUBE_URL`, correct?

Reply:
Only the hypercube image is different, cloud-controller-manager remains the same

Reply:
The point is, this implementation doesn't actually pre-pull the image in the Azure Stack case. The if should be:

```
if [[ $KUBERNETES_VERSION == *"azs"* ]]; then
  HYPERKUBE_URL="msazurestackdocker/hyperkube-amd64:v${KUBERNETES_VERSION}"
else
  HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
fi
```

Unless I'm missing something.

Reply:
Function `extractHyperkube` calls `pullContainerImage $CLI_TOOL ${HYPERKUBE_URL}`.

That clears your concern, right?